draft-ietf-httpbis-priority.md

```diff
@@ -523,12 +523,12 @@ HTTP/2 PRIORITY_UPDATE Frame {
 {: #fig-h2-reprioritization-frame title="HTTP/2 PRIORITY_UPDATE Frame Payload"}
 
 The Length, Type, Unused Flag(s), Reserved, and Stream Identifier fields are
-described in {{Section 4 of HTTP2}}. The frame payload of PRIORITY_UPDATE
-frame payload contains the following additional fields:
+described in {{Section 4 of HTTP2}}. The frame payload of PRIORITY_UPDATE frame
+contains the following additional fields:
 
 Reserved:
-: A reserved 1-bit field. The semantics of this bit are undefined, and the bit
-  MUST remain unset (0x0) when sending and MUST be ignored when receiving.
+: A reserved 1-bit field. The semantics of this bit is undefined. It MUST remain
+  unset (0x0) when sending and MUST be ignored when receiving.
```

Comment:
the current text is verbatim how H2 defines it in https://httpwg.org/http2-spec/draft-ietf-httpbis-http2bis.html#section-4.1-4.8.1

The first part should remain "semantics ... are" due to grammar. The second part of the change is ok by me

Reply:
New Oxford American Dictionary says that the word "semantics" is "usually treated as singular," but I'm happy to not be American.

Reply:
Fixed in d3d71ba.